src/scripts/underground/UndergroundUpgrade.ts

```diff
@@ -53,3 +53,10 @@ Underground.upgradeList.push(
         GameHelper.createArray(0, 2, 1)
     )
 );
+
+Underground.upgradeList.push(
+    new UndergroundUpgrade(Underground.Upgrades.Bomb_Efficiency, 'Bomb Efficiency', 5,
+        AmountFactory.createArray(GameHelper.createArray(50, 500, 50), GameConstants.Currency.diamond),
+        GameHelper.createArray(0, 10, 2)
```

Comment:
There are only 5 steps in the bonus array, but the cost array has 10 steps (max cost is only 250, not 500). Not really a bug, but makes me think one of these numbers maybe isn't what you intended?

Reply:
Ah that makes sense, I wasn't really sure what the array did, as it's not too self explanatory 😅 
What is the first number used for?
Also what are the first/last numbers in the Amount Factory?

Reply:
Yeah, it is `start, end, increment`, so you either want your bonus array to be built with `(0, 10, 1)` for 10 steps, or the amount array built from `(50, 250, 50)` to just make the 5 steps we actually have a bonus for.